client-sdk/ts-web/core/package-lock.json

```diff
@@ -0,0 +1,5752 @@
+{
```

Comment:
We're on the npm 7 boat, where there's a shared package-lock.json one directory level out. Would you be able to use npm 7 to run `npm install` out in `client-sdk/ts-web` and update that package-lock.json instead of adding this file?

Reply:
Done